orchestrator.py

```diff
@@ -143,7 +157,9 @@ def on_connect(self, client, userdata, flags, rc):
         client.subscribe(str(topics.Camera.set_overlay))
         client.subscribe(str(topics.Camera.get_overlays))
         client.subscribe(str(topics.WirelessModule.all().module))
-        client.subscribe(str(topics.Camera.flip_video_feed / self.device))
+        client.subscribe(
+            topics.Camera.flip_video_feed + "/" + self.device
+        )  # operator overload wasn't working on rpi, quick fix
```

Comment:
@kdav108, was there a recent change in the common repo that could have caused this? When looking at the issue with Arosh yesterday, it looked like the topics were returning strings rather than `Topic` objects, meaning we couldn't call `__truediv__` or `matches` on them.

We should add unit tests to make sure this sort of thing doesn't break :) Perhaps you want to hand this to Arosh

Reply:
yep that's the issue, common repo treats them as string because we use the `type()` method in python to generate the classes. That's my bad, I'll see if I can easily fix it and still use `type()`.